cmd/openfga/openfga.go

```diff
@@ -34,11 +35,20 @@ var (
 )
 
 type service struct {
-	openFgaServer *server.Server
+	server        *server.Server
 	datastore     storage.OpenFGADatastore
 	authenticator authentication.Authenticator
 }
 
+func (s *service) Close(ctx context.Context) error {
+	s.authenticator.Close()
+
+	return multierr.Combine(
```

Comment:
Let's not bring a whole other external package in just for this.

```
if err := s.server.Close(); err != nil {
    return err
}

if err := s.datastore.Close(ctx); err != nil {
    return err
}
```

is fine.

Reply:
I just realized that server.Close doesn't need an error, so I updated that instead.